src/components/Mandal/view/Todo.js

```diff
@@ -0,0 +1,162 @@
+      days,
+    },
+  });
+  return result;
+};
+
+const saveTodo = async (todoId, date, repetition) => {
+  const result = await axios.post(`/api/todos/${todoId}`, { date, repetition });
+};
+
+export default function Todo({ id }) {
+  const [isLoading, setIsLoading] = useState(true);
+  const [todos, setTodos] = useState();
+  const [currentDate, setCurrentDate] = useState(new Date());
+  const [isRepeat, setIsRepeat] = useState(false);
+  const [repetition, setRepetition] = useState(EVERY_DAY);
+  const [duration, setDuration] = useState(1);
+  const loginState = useSelector(state => state.user.loginSucceed);
+
+  const getTodosOfDay = async () => {
+    const res = await getUsersTodo(currentDate, ONE_DAY);
+    if (res.data.result === "error") {
+      return;
+    }
+
+    setTodos(res.data.result[format(currentDate, "yyyy-MM-dd")]);
+    setIsLoading(false);
+  };
+
+  useEffect(() => {
+    if (loginState) {
+      getTodosOfDay();
+    }
+  }, [loginState, currentDate]);
+
+  const handleDateChange = event => {
+    setCurrentDate(new Date(event.target.value));
+  };
+
+  const showTodosInDate = () => {
+    if (!todos) {
+      return <div> 저장된 일이 없습니다.</div>;
+    }
+    return todos?.map((todo, i) => {
+      return (
+        <div key={i}>
+          {todo.title} {todo.isComplete}
+        </div>
+      );
+    });
+  };
+
+  const handleRepetitionCheck = event => {
+    event.target.checked ? setIsRepeat(true) : setIsRepeat(false);
+  };
+
+  const handleRepetitionButton = event => {
+    setRepetition(event.target.value);
+  };
+
+  const handleSelectButton = event => {
+    setDuration(event.target.value);
+  };
+
+  return (
+    <>
+      <div>할일 저장하기</div> <hr />
+      <BodyContainer>
+        <ContentContainer>
+          <input
+            type="date"
+            value={format(currentDate, "yyyy-MM-dd")}
+            onChange={handleDateChange}
+          />
+          <input
+            type="checkbox"
+            id="repetitionCheckBox"
+            name="repetitionCheckBox"
+            onChange={handleRepetitionCheck}
+          />
+          <label htmlFor="repetitionCheckBox"> 반복</label>
+          {isRepeat && (
+            <RadioButtonContainer>
+              <RadioButton
+                handleOnChange={handleRepetitionButton}
+                options={REPETITION}
+              />
+              <RadioButton
+                handleOnChange={handleSelectButton}
+                options={DURATION}
+              />
+            </RadioButtonContainer>
+          )}
+        </ContentContainer>
+        <ContentContainer>
+          <div>저장된 할 일들</div>
+          {isLoading ? <div>로딩 중</div> : <div>{showTodosInDate()}</div>}
+          <input
+            type="button"
+            value="Add"
+            onClick={() =>
+              saveTodo(id, currentDate, {
+                isRepeat,
+                type: repetition,
+                duration,
+              })
+            }
```

Comment:
할 일을 추가해주는 버튼 잘 만들어주셨네요👍 onClick도 다른 부분과 통일성있게 따로 함수로 빼주면 어떨까요??
```suggestion
            onClick={handleAddButtonClick}
```